### ISSUE TITLE
Convert blenderbot checkpoint to tensorflow (TF)

### ISSUE THREAD

**Issue by sooftware** [huggingface/transformers#11741]
Hi! Thank you for a great project.  
I wonder if I can convert blenderbot checkpoint to tensorflow.  
If I can, how I convert checkpoint? Give me some comment.

**Comment by sooftware** [comment]
@patrickvonplaten Can you help me?

**Comment by sooftware** [comment]
I convert Parl-AI's checkpoint to huggingface using `convert_blenderbot_original_pytorch_checkpoint_to_pytorch.py`. 
And I convert pytorch checkpoint to tf checkpoint using `convert_pytorch_checkpoint_to_tf2.py`.  
  
If there is something wrong, please comment.

**Comment by patrickvonplaten** [comment]
Hey @sooftware,

Could you add a code snippet you are trying to execute here? E.g. which checkpoint do you want to convert exactly?

**Comment by sooftware** [comment]
Hi @patrickvonplaten !!  
  
I want to convert Parl-AI's blanerbot (3B, 9B) models.  
I tried to convert Parl-AI to huggingface checkpoint by `convert_blenderbot_original_pytorch_checkpoint_to_pytorch.py`.  
  
I change some keys.  
  
```python
def rename_layernorm_keys(sd):    
    keys = [        
        "model.encoder.layernorm_embedding.weight",        
        "model.encoder.layernorm_embedding.bias",        
        "model.decoder.layernorm_embedding.weight",        
        "model.decoder.layernorm_embedding.bias",    
    ]
```
  
Ex) `model.encoder.layernorm_embedding.weight` => `encoder.norm_embeddings.weight`. 
  
And I got a config file by `wget https://huggingface.co/facebook/blenderbot-3B/resolve/main/config.json`.  
   
Next, I tried to convert huggingface pytorch checkpoint to tensorflow checkpoint by `huggingface convert_pytorch_to_tf.py`.  
I set `model_type` by `bart`.
